api/backend.go

```diff
@@ -233,7 +233,9 @@ func (b *StatusBackend) CallPrivateRPC(inputJSON string) string {
 
 // SendTransaction creates a new transaction and waits until it's complete.
 func (b *StatusBackend) SendTransaction(ctx context.Context, args transactions.SendTxArgs) (hash gethcommon.Hash, err error) {
-	return b.transactor.SendTransaction(ctx, args)
+	hash, err = b.transactor.SendTransaction(ctx, args)
+	// TODO (seb): trigger the newPendingTransaction event here.
```

Comment:
@adambabik @mandrigin What is a good way to trigger the `newPendingTransactionEvent` from here?

Right now its set up so that I should be able to call `api.transactionSentToUpstreamEvent.TransactionSentToUpstream()` from `rpcfilters.PublicAPI` but I need to figure out how to get access to `rpcFiltersService` from backend.go. Right now its appended to the services list in the statusNode initialization inside `backend.startNode`, but I don't see how to access that instance.

Would it be reasonable to add `rpcFiltersService` to the `StatusBackend` struct?